src/index.js

```diff
@@ -30,30 +30,19 @@ const Container = ({ options }) =>
 const onfidoRender = (options, el, merge) =>
   render( <Container options={options}/>, el, merge)
 
-const stripOneCapture = ({blob, documentType, onfidoId, side}) => {
-  const capture = {id: onfidoId, blob}
-  if (documentType) capture.documentType = documentType
-  if (side) capture.side = side
-  return capture
-}
-
-const stripCapturesHash = captures => mapValues(captures,
-  capture => capture ? stripOneCapture(capture) : null)
+const mapCaptures = captures => mapValues(captures, capture => capture)
 
-const getCaptures = () => mapKeys(stripCapturesHash(events.getCaptures()), key => key + 'Capture')
+const getCapturesKeys = () => mapKeys(mapCaptures(events.getCaptures()), key => key + 'Capture')
 
 function bindEvents (options) {
   const eventListenersMap = {
-    documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
-    documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
-    faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
     complete: () => {
-      const captures = getCaptures();
+      const captures = getCapturesKeys();
```

Comment:
Why do we need to do this? What are we sending to woopra?

Reply:
We are sending an object to confirm which captures have been uploaded, it looks something like this `{faceCapture: true, documentCapture: true, documentBackCapture: false}`

Reply:
Ok, but that's not reliable in a cross device flow. Do we already track this information at the point of upload?

Do we need the concept of a confirmed capture anymore?

Reply:
Yes, good point. I don't think we need the concept of confirmed captures because we don't have those events anymore. And therefore I don't see the point of sending those info to woopra either...